projects/js-packages/publicize-components/src/components/connection-management/connection-status.tsx

```diff
@@ -26,7 +29,14 @@ export function ConnectionStatus( { connection, service }: ConnectionStatusProps
 			<span className="description">
 				{ service
 					? __( 'There is an issue with this connection.', 'jetpack' )
-					: _x( 'This platform is no longer supported.', '', 'jetpack' ) }
+					: createInterpolateElement(
+							'This platform is no longer supported. Use <link>Manual Sharing</link> instead.',
```

Comment:
What if LinkedIn is broken and we don't support quick sharing for it? Should we perhaps leave this bit untouched and just mention quick share outside of the modal?

Reply:
It will reach that alternate block only if the service is `undefined`, i.e. it's not supported anymore. If the connection is broken, it won't reach that block.